### PR TITLE
Reorganize FAQ and update the content

### PR DESCRIPTION
**Pull request by lilin90**
@CaitinChen @dbaoutdo PTAL. Via the following 3 PRs: 
- https://github.com/pingcap/docs-cn/pull/545
- https://github.com/pingcap/docs-cn/pull/546
- https://github.com/pingcap/docs-cn/pull/574

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -7,105 +7,281 @@ category: faq
...
+#### Is it easy to use TiDB?
 
-TiDB works as the SQL layer and TiKV works as the Key-Value layer. TiDB is mainly responsible for parsing SQL, specifying query plan, and generating executor while TiKV is to store the actual data and works as the storage engine.
+Yes, it is. When all the required services are started, you can use TiDB as easily as a MySQL server. You can replace MySQL with TiDB to power your applications without changing a single line of code in most cases. You can also manage TiDB using the popular MySQL management tools.
 
-TiDB provides TiKV the SQL enablement and turns TiKV into a NewSQL database. TiDB and TiKV work together to be as scalable as a NoSQL database while maintains the ACID transactions of a relational database.
+#### How is TiDB compatible with MySQL?
 
-#### What does Placement Driver (PD) do?
+Currently, TiDB supports the majority of MySQL 5.7 syntax, but does not support trigger, stored procedures, user-defined functions, and foreign keys. For more details, see [Compatibility with MySQL](sql/mysql-compatibility.md).
```
Comment (CaitinChen) [inline FAQ.md]: triggers

File: FAQ.md [hunk 2]
```diff
@@ -643,10 +715,87 @@ Recommendations:
...
+Use `admin show ddl` to view the current job of adding an index.
+
+#### How to view the DDL job?
+
+- `admin show ddl`: to view the running DDL job
+- `admin show ddl jobs`: to view all the results in the current DDL job queue (including tasks that are running and waiting to run) and the last ten results in the completed DDL job queue
+
+#### Does TiDB support CBO (Cost-Based Optimization)? If yes, to what extent?
+
+Yes. TiDB uses the cost-based optimizer. The cost model and statistics are constantly optimized. Besides, TiDB also supports correlation algorithms like Hash join and soft merge.
```
Comment (CaitinChen) [inline FAQ.md]: **h**ash

File: FAQ.md [hunk 3]
```diff
@@ -586,52 +655,55 @@ As distributed transactions need to conduct two-phase commit and the bottom laye
...
+
+If the amount of data that needs to be deleted at a time is very large, this loop method will get slower and slower because each deletion traverses backward. After deleting the previous data, lots of deleted flags will remain for a short period (then all will be Garbage Collected) and influence the following Delete statement. If possible, it is recommended to refine the Where condition. See [details in TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/#write).
+
+#### How to improve the data loading speed in TiDB?
+
+- Currently Lightning is in development for distributed data import. It should be noted that the data import process does not perform a complete transaction process for performance reasons. Therefore, the ACID constraint of the data being imported during the import process cannot be guaranteed. The ACID constraint of the imported data can only be guaranteed after the entire import process ends. Therefore, the applicable scenarios mainly include importing new data (such as a new table or a new index) or the full backup and restoring (Truncate the original table and then import data).
```
Comment (CaitinChen) [inline FAQ.md]: **t**runcate

File: FAQ.md [hunk 4]
```diff
@@ -586,52 +655,55 @@ As distributed transactions need to conduct two-phase commit and the bottom laye
...
 
-### SQL optimization
+#### What is the most efficient way of deleting data?
 
-#### How to view the query execution plan of TiDB?
+When deleting a large amount of data, it is recommended to use `Delete * from t where xx limit 5000;`. It deletes through the loop and uses `Affected Rows == 0` as a condition to end the loop, so as not to exceed the limit of transaction size. With the prerequisite of meeting business filtering logic, it is recommended to add a strong filter index column or directly use the primary key to select the range, such as `id >= 5000*n+m and id < 5000*(n+1)+m`.
+
+If the amount of data that needs to be deleted at a time is very large, this loop method will get slower and slower because each deletion traverses backward. After deleting the previous data, lots of deleted flags will remain for a short period (then all will be Garbage Collected) and influence the following Delete statement. If possible, it is recommended to refine the Where condition. See [details in TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/#write).
```
Comment (CaitinChen) [inline FAQ.md]: (then all will be Garbage Collected) → (then all will be dealt with by Garbage Collection)

**Comment by dbaoutdo** [comment]
LGTM

**Pull request merged**